src/assets/discrete-assets.md

```diff
@@ -75,4 +75,8 @@ curl -X POST "https://service.centrapay.com/api/external-assets" \
 
 | Status | Code                    | Description                                         |
 |:-------|:------------------------|:----------------------------------------------------|
-| 403    | {% break _ FORBIDDEN %} | Invalid external id or pin or asset already exists. |
+| 500    | SERVICE_UNAVAILABLE     | Upstream server unavailable.                        |
```

Comment:
We don't need to document 5xx errors as they can always be handled the same way.

Reply:
Cool, I'll remove this.